enum.h

```diff
@@ -349,7 +351,7 @@ BETTER_ENUMS_CONSTEXPR_ static T* _or_null(optional<T*> maybe)
 
 template <typename T, typename U>
 BETTER_ENUMS_CONSTEXPR_ U
-continue_with(T ignored BETTER_ENUMS_UNUSED, U value) { return value; }
+continue_with(T BETTER_ENUMS_UNUSED_WRAPPER(ignored) BETTER_ENUMS_UNUSED, U value) { return value; }
```

Comment:
Alternative PR is welcome. I didn't merge this mainly because of lack of time to port this PR to clang (see [Travis failures](https://travis-ci.org/aantron/better-enums/builds/220451200)), easy as that should be :/